website1/src/scss/sections/_what-is-bad.scss

```diff
@@ -0,0 +1,74 @@
+.what-is-bad {
+  &-wrapper {
+    width: 100%;
+    padding-bottom: 100px;
+    background-color: $white-back;
+  }
+
+  &__title {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__remedies {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 13px 20px;
+    justify-content: center;
+    padding-top: 33px;
+  }
+}
+
+.button-prevent {
+  margin-top: 75px;
+}
+
+.prevent {
+  padding: 9px 74px;
+}
+
+.remedies-box {
+  display: flex;
+  flex-direction: column;
+  width: 300px;
+  height: 300px;
+  background-color: $white;
+  padding: 33px 25px;
+
+  &__description {
+    margin-top: 28px;
+    font-weight: 300;
+    line-height: 159.02%;
+    text-align: center;
+  }
+
+  &__icon {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+}
+
+.icon {
+  &__img {
+    position: relative;
+    z-index: 2;
```

Comment:
`z-index` - important values, move to variables because you need to see all `z-index` in one file.